src/components/details-card/details-card.tsx

```diff
@@ -0,0 +1,126 @@
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import * as React from 'react';
+import * as _ from 'lodash';
+import { Link, BrowserRouter as Router, } from 'react-router-dom';
+import {
+    Card,
+    CardBody,
+    CardHeader,
+    CardTitle,
+  } from "@patternfly/react-core";
+import { Base64 } from 'js-base64';
+import {
+    useK8sWatchResource,
+} from "@console/dynamic-plugin-sdk/api";
+import {
+  DetailItem,
+  DetailsBody,
+} from "@console/dynamic-plugin-sdk/provisional";
+import {ExternalLink} from './Link';
+import { 
+  StorageInstanceKind, 
+  K8sKind,
+  SecretKind
+ } from '../../types';
+import { 
+  getEndpoint,  
+  getIBMStorageODFVersion,
+  resourcePathFromModel,
+  } from '../../selectors';
+import {
+  ClusterServiceVersionModel,
+} from '../../models';
+import {
+  GetFlashSystemResource, 
+  SubscriptionResource,
+  GetSecretResource 
+} from '../../constants/resources'
+
+const DetailsCard: React.FC<any> = (props) => {
+  const flashClusterResource = GetFlashSystemResource(props?.match?.params?.name, props?.match?.params?.namespace);
+  const [data, flashsystemloaded, flashsystemloadError] = useK8sWatchResource<StorageInstanceKind>(flashClusterResource);
+  const [subscriptions, subscriptionloaded, subscriptionloadError] = useK8sWatchResource<K8sKind[]>(SubscriptionResource);
+
+  const stoData = data?.[0];
+  const flashSecretResource = GetSecretResource(stoData?.metadata?.name, stoData?.metadata?.namespace);
+  const [secret, secretloaded, secretloadError] = useK8sWatchResource<SecretKind>(flashSecretResource);
+  const endpointAddress = secretloaded && !secretloadError ? Base64.decode(getEndpoint(secret)) : '';
+
+  const flashOperatorVersion = getIBMStorageODFVersion(subscriptions);
+  const operatorPath = `${resourcePathFromModel(
+    ClusterServiceVersionModel,
+    flashOperatorVersion,
+    stoData?.metadata.namespace,
+  )}`;
+
+  return (
+    <Card className="co-dashboard-card co-dashboard-card--gradient">
+    <CardHeader className="co-dashboard-card__header">
+      <CardTitle className="co-dashboard-card__title">Details</CardTitle>
+    </CardHeader>
+    <CardBody className="co-dashboard-card__body">
+       <DetailsBody>
+          <DetailItem
+            key="operator-name"
+            title="Operator Name"
+            error={!!subscriptionloadError }
+            isLoading={!subscriptionloaded }
+          >
+            <Router> <Link to={operatorPath} onClick={() => window.location.href=operatorPath}>OpenShift Data Foundation - IBM</Link> </Router>
```

Comment:
Why onClick() ?

Reply:
I find the the Link from react-router-dom does not jump to destination address in dynamic-plugin mode. So as a workaround, I add the force onClick here to goto the path.

Reply:
ack